integration/tests/cook/test_cli.py

```diff
@@ -872,7 +872,9 @@ def entry(name):
     def test_show_progress_message(self):
         executor = util.get_job_executor_type(self.cook_url)
         cp, uuids = cli.submit('echo "progress: 99 We\'re so close!"', self.cook_url,
-                               submit_flags=f'--executor {executor}')
+                               submit_flags=f'--executor {executor} '
+                                            f'--env EXECUTOR_PROGRESS_OUTPUT_FILE_ENV=PROGRESS_OUTPUT_FILE '
+                                            f'--env PROGRESS_OUTPUT_FILE=stdout')
```

Comment:
The second and third lines don't need to be f-strings

Reply:
Done.